applications/serial_lte_modem/src/slm_at_tcp_proxy.c

```diff
@@ -568,7 +568,7 @@ static int tcpsvr_input(int infd)
 			      K_NO_WAIT);
 	} else {
 		k_timer_stop(&conn_timer);
-		ret = recv(fds[infd].fd, rx_data, sizeof(rx_data), 0);
+		ret = recv(fds[infd].fd, (void *)rx_data, sizeof(rx_data), 0);
```

Comment:
What's the exact complaint related to this change? `void *` argument should accept any pointer w/o a warning, it's a common pattern across the codebase.

Reply:
It's a change that I'm not sure, either. Coverity complains that the buffer will have "tainted" data, as we discussed yesterday. I'm trying this void pointer cast (from uint8_t pointer) to see whether it could please Coverity or not.

Reply:
I doubt a typecast would resolve the "tainted" data problem to be honest.

Reply:
Neither do I, don't know how to please Coverity for this complain, yet